src/OrbitGl/CaptureStatsTest.cpp

```diff
@@ -10,18 +10,14 @@
 #include "TextBox.h"
 
 TextBox* CreateScope(int32_t pid, int32_t tid, int32_t cpu, uint64_t start, uint64_t end) {
-  const size_t kNumScopes = 1024;
-  static TextBox scope_buffer[kNumScopes];
-  static size_t index = 0;
-  EXPECT_TRUE(index < kNumScopes - 2);
```

Comment:
I know that we are inside a test, but we are not having this invariant anymore or simply we are not checking it?

Reply:
As far as I understand, this was just a limit posed by using an array, but a vector can just be used instead.

Reply:
At the same time now a test fails on Windows in this file, so let's see.

Reply:
Rearranged this code a bit and switched to a list.